.github/workflows/PR.yml

```diff
@@ -91,5 +91,11 @@ jobs:
           name: error-report
           path: build-reports.zip
 
+      - name: Publish Test Report
+        uses: mikepenz/action-junit-report@v3
+        if: github.repository_owner == 'cashapp' # always run even if the previous step fails
```

Comment:
Comment doesn't match the key/value pair. Are we missing `if: ${{ always() }}`?

Reply:
Or is that what you replaced?

Reply:
Ah yeah that got replaced. 

I guess the comment should be:

```
# always run even if the previous step fails if the source is from the cashapp org
```

Reply:
PR welcome! :D